app/models/models_test.go

```diff
@@ -255,6 +255,42 @@ func TestListQueuesRequest_SetAttributesFromForm_invalid_max_results(t *testing.
 	assert.Equal(t, "queue-name-prefix", lqr.QueueNamePrefix)
 }
 
+func TestSendMessageRequest_SetAttributesFromForm_success(t *testing.T) {
+	form := url.Values{}
+	form.Add("MessageAttribute.1.Name", "Attr1")
+	form.Add("MessageAttribute.1.Value.DataType", "String")
+	form.Add("MessageAttribute.1.Value.StringValue", "Value1")
+	form.Add("MessageAttribute.2.Name", "Attr2")
+	form.Add("MessageAttribute.2.Value.DataType", "String")
+	form.Add("MessageAttribute.2.Value.BinaryValue", "VmFsdWUy")
+	form.Add("MessageAttribute.3.Name", "")
+	form.Add("MessageAttribute.3.Value.DataType", "String")
+	form.Add("MessageAttribute.3.Value.StringValue", "Value")
+	form.Add("MessageAttribute.4.Name", "Attr4")
+	form.Add("MessageAttribute.4.Value.DataType", "")
+	form.Add("MessageAttribute.4.Value.StringValue", "Value4")
```

Comment:
Would it make sense to make at least one of these fields a `BinaryValue`?

Reply:
The test for "Attr2" is that 👍